docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Hardware Key Support (Preview)
+description: Hardware Key Support
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Hardware key Support is currently in Preview mode.
+</Admonition>
+
+By default, `tsh`, Teleport Connect, and other Teleport clients store a user's login session directly on their filesystem. If a user's filesystem is compromised, any of their active Teleport login sessions would also be compromised.
+
+[Per-session MFA](per-session-mfa.mdx) provides a way to require an MFA presence check with a user's login session, but this check is only used for starting sessions with Teleport services (SSH, Kube, DB, etc.). This means that non-session actions, like administrative commands run with `tctl`, can still be run with an exfiltrated login session. 
+
+To prevent these types of exfiltration attacks altogether, Teleport clients support hardware-based private keys. Unlike disk-based private keys, hardware-based private keys are generated and stored directly on a hardware device and are impossible to export. With this features, a user's login session is only functional with access to the user's hardware device.
+
+Additionally, this feature can be configured to require touch for every Teleport request, including non-session requests. With touch required, hardware key support provides better security in comparison to our per-session MFA offering.
+
+<Admonition type="note" title="PIV support">
+  Hardware key support requires users to use a PIV-compatible hardware key. Currently, this feature is only guarenteed to support YubiKey series 5+.
```

Comment:
Perhaps add a line here about what this process will do to any existing keys/certs in the PIV store of the key.